tests/e2e/login.e2e.spec.ts

```diff
@@ -1,5 +1,7 @@
 import { expect, test } from "@nuxt/test-utils/playwright";
 
+test.setTimeout(60000);
```

Comment:
Do both of the tests require an increased timeout? If not, can we increase it only for the one test that fails before merging?

Reply:
Hello @yurijmikhalevich!
yes, both login-related tests fail due to exceeding the default timeout.

![image](https://github.com/user-attachments/assets/f49f7be7-1a7f-4976-b964-6996070343a1)

Reply:
@arilloid, I see. It's probably related to the time it takes to load and render the GameScreen until the page stabilizes.